docs/SUMMARY.md

```diff
@@ -135,6 +130,11 @@
     * [Monorepo Python Development](contributing-to-airbyte/building-new-connector/monorepo-python-development.md)
     * [Testing Connectors](contributing-to-airbyte/building-new-connector/testing-connectors.md)
     * [Source Connector Acceptance Tests](contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+    * [Tutorials](contributing-to-airbyte/tutorials/README.md)
```

Comment:
@avaidyanatha there is another folder Tutorials inside **Connector Development Kit \(Python\)** path, should it merge too?

Reply:
Personally I wanted to keep them separate since they are targeted at very different audiences but I do see the argument for combining them, what do you feel?

Reply:
For now can be good have a section exclusive for CDK, in the future this will be only the default way to contribute to Airbyte